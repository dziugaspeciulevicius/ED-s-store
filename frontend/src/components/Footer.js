import React from "react";
import "../sass/components/Footer.styles.scss";

const Footer = () => {
  return (
    <div class="footer">
      <div class="container">
        <div class="row">
          <div class="col-lg-4 col-sm-4 col-xs-12">
            <div class="single_footer">
              <h4>Services</h4>
              <ul>
                <li>
                  <a href="#">Lorem Ipsum</a>
                </li>
                <li>
                  <a href="#">Simply dummy text</a>
                </li>
                <li>
                  <a href="#">The printing and typesetting </a>
                </li>
                <li>
                  <a href="#">Standard dummy text</a>
                </li>
                <li>
                  <a href="#">Type specimen book</a>
                </li>
              </ul>
            </div>
          </div>
          <div class="col-md-4 col-sm-4 col-xs-12">
            <div class="single_footer single_footer_address">
              <h4>Page Link</h4>
              <ul>
                <li>
                  <a href="#">Lorem Ipsum</a>
                </li>
                <li>
                  <a href="#">Simply dummy text</a>
                </li>
                <li>
                  <a href="#">The printing and typesetting </a>
                </li>
                <li>
                  <a href="#">Standard dummy text</a>
                </li>
                <li>
                  <a href="#">Type specimen book</a>
                </li>
              </ul>
            </div>
          </div>
          <div class="col-md-4 col-sm-4 col-xs-12">
              <div class="social_profile">
                <ul>
                  <li>
                    <a href="https://www.linkedin.com/in/dziugaspeciulevicius/">
                      <i class="fab fa-linkedin"></i>
                    </a>
                  </li>
                  <li>
                    <a href="https://github.com/dziugaspeciulevicius">
                      <i class="fab fa-github"></i>
                    </a>
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </div>
  );
};

export default Footer;
