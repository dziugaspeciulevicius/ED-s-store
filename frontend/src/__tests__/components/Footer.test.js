
import React from "react";
import { shallow } from "enzyme";
import "../../setupTests";
import Footer from "../../components/Footer.component";

describe("Footer tests", () => {
    it("Renders <Footer /> component correctly", () => {
      // shallow creates instance of a component
      expect(shallow(<Footer />)).toMatchSnapshot();
    });
});
