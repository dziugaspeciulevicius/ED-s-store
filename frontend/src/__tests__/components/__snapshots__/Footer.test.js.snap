// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests Renders <Footer /> component correctly 1`] = `ShallowWrapper {}`;
